.gitignore

```diff
@@ -2,3 +2,4 @@ doc/
 node_modules/
 tmp/
 npm-debug.log
+.project
```

Comment:
.project is a user thing, it should not be commited to the project's gitignore but rather in your global `~/.gitignore` file.